codalab/worker/bundle_manager.py

```diff
@@ -311,6 +311,12 @@ def _schedule_run_bundles_on_workers(self, workers, user_owned):
             if user_owned:
```

Comment:
Does this loop go through the runs in the order in which they are created?  I assume yes, but probably should be make explicit.

Reply:
Looking into the `batch_get_bundles` function, the bundles are sorted by `id`, which I'm guessing is the same as creation order.

I think ideally there would be a separate scheduler component that would take in this list and sort  it with a more involved logic, but I'm not sure this PR is the right place to do this, I think just to keep the PR simple, I shouldn't change this part of the code.

Reply:
Yeah, let's leave the code as is, but put a comment documenting the limitations and the need for a real scheduler.